### PR TITLE
Show meetings in user's correct timezone 

### PR DESCRIPTION
**Pull request by oliverguenther**
# Ticket
https://community.openproject.org/work_packages/57078

# What are you trying to accomplish?
Use the correct timezone format for displaying meetings (only objects in calendars with times right now), so that they match the meeting time formatted in the backend

### CODE REVIEW

File: modules/calendar/spec/features/calendar_widget_spec.rb [hunk 1]
```diff
@@ -50,15 +50,19 @@
   let(:wp_full_view) { Pages::FullWorkPackage.new(work_package, project) }
   let(:calendar) { Pages::Calendar.new project }
 
-  current_user do
+  shared_let(:current_user) do
```
Comment (cbliard) [inline modules/calendar/spec/features/calendar_widget_spec.rb:53]: There is now a `shared_current_user` which does the same as `current_user`, but with `shared_let`.

**Review by cbliard — APPROVED** [review]
Nice 👏

File: modules/calendar/spec/features/calendar_widget_spec.rb (hunk 1 above)
Comment (oliverguenther) [reply modules/calendar/spec/features/calendar_widget_spec.rb:53]: The problem with these helpers is that they create a new `before` block, which changes the execution. I found this is weird if you multiple of these

Comment (cbliard) [reply modules/calendar/spec/features/calendar_widget_spec.rb:53]: good point.

**Pull request merged**